source/codegen/validate_examples.py

```diff
@@ -79,7 +79,7 @@ def _validate_examples(
             _system(f"poetry run black --check --line-length 100 {dir}")
 
             print(f" -> Running mypy")
-            _system(f"poetry run mypy {dir} --check-untyped-defs")
+            _system(f"poetry run mypy {dir} --check-untyped-defs --ignore-missing-imports")
```

Comment:
I did this temporarily to avoid all the type errors related to `numpy` or `matplotlib`. 

But I wanted to ask, should I leave this in and update the CI to run for all examples except DAQmx? Is there a downside to doing this?

Reply:
Yeah, I think this is a pretty good tradeoff. The worst case is a typo in some other `import` could cause us to silently skip type checking on a lot of files. Or a mistake in our setup code that causes the protoc types to be missing. So I think it's worth trying to remove this flag, but this change as-is is better than the current state.